contribs/drt/src/main/java/org/matsim/contrib/drt/run/DrtConfigGroup.java

```diff
@@ -184,6 +184,15 @@ public enum OperationalScheme {
 	@PositiveOrZero
 	public double advanceRequestPlanningHorizon = 0; // beta-feature; planning horizon for advance (prebooked) requests
 
+	@Parameter
+	@Comment(
+			"Time before reaching a planned dropoff from which it is not allowed to insert new detours for new requests. I.e.," +
+					" if set to 180, then a vehicle will not divert to pickup a new passenger once a boarded passenger is only " +
+					"3 minutes away from her destination, even though her time window would allow it." +
+					" Delayed detour just before arrival are usually perceived very negatively.")
+	@PositiveOrZero // used only for stopbased DRT scheme
+	public double fixedApproachTime = 0;// [m];
```

Comment:
`fixedApproachTime` is too general. It will be very hard to memorise what it means. I think we need a name that would be more expressive.

Reply:
How about "allowDetourBeforeArrivalThreshold"?